tests/regressions/execution_tree/allow_return_nil_432.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 Parsa Amini
+// Copyright (c) 2018 Hartmut Kaiser
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+// Fixing #432: Cannot return nil in PhySL
+
+#include <phylanx/phylanx.hpp>
+
+#include <hpx/hpx_init.hpp>
+#include <hpx/include/iostreams.hpp>
+#include <hpx/util/lightweight_test.hpp>
+
+#include <string>
+#include <sstream>
+
+std::string const code = R"(
+    define(f, block(
+        nil
+    ))
+    cout(f())
+)";
+
```

Comment:
Could you please add another test where nil is not the only statement inside the block, please?

Reply:
Done